brave/brave-ios
# Fix #890 - Stop responding to and opening firefox:// URL schemes.

## Submitter Checklist:



## Test Plan:


1. Install Firefox for iOS
2. Install Brave
3. Go to Firefox's "Today" widget and select "New Tab" or "New Private Tab"
4. No longer get prompted to "Open in 'Brave'" (should automatically open in Firefox for iOS)

### Screenshots:




## Reviewer Checklist:
